ansible/roles/nginx_vhost/tasks/main.yml

```diff
@@ -354,6 +372,16 @@
   tags:
     - nginx_vhost
 
+- name: add cors fragment if required
+  template:
+    src: "fragment_74_location_cors.j2"
+    dest: "{{nginx_conf_dir}}/vhost_fragments/{{hostname}}/https_70_location_{{ item.sort_label | default(item.path | basename) }}_74_cors"
```

Comment:
The destination is named `http_70_location`. I think that should be `http_74_location` to ensure the fragment numbering matches the template numbering for debugging purposes.

Reply:
the destination is `https_70_location_1_74_cors` so this is the result:
```
https_70_location_1_70_start
https_70_location_1_73_content
https_70_location_1_74_cors
https_70_location_1_75_end
```